tutorials/navigation/navigation_using_navigationobstacles.rst

```diff
@@ -179,6 +221,33 @@ For static use an array of ``vertices`` is required.
     # Enable the obstacle.
     NavigationServer2D.obstacle_set_avoidance_enabled(new_obstacle_rid, true)
 
+ .. code-tab:: csharp 2D C#
+
+    Vector2 globalPosition = GlobalPosition;
+
```

Comment:
This variable doesn't seem needed, it's only used once so we can just use the `GlobalPosition` property directly.

```suggestion
```